src/Evolution/Systems/GrMhd/GhValenciaDivClean/Tags.hpp

```diff
@@ -0,0 +1,34 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "Evolution/Systems/GrMhd/GhValenciaDivClean/TagsDeclarations.hpp"
+#include "Evolution/Tags.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+namespace grmhd {
+namespace GhValenciaDivClean {
```

Comment:
clang-tidy will presumably tell you to combine these two lines.  (And again in the declarations file.)